lib/command/list.rb

```diff
@@ -148,16 +148,18 @@ def header
     end
 
     def view_date_type
-      @options["general-lastup"] ? "general_lastup" : "last_update"
+      @_view_data_type ||= @options["general-lastup"] ? "general_lastup" : "last_update"
     end
 
-    def output_list(novels)
-      now = Time.now
-      today = now.strftime("%y/%m/%d")
+    def now
+      @now ||= Time.now
+    end
+
+    def output_list(novels, limit)
```

Comment:
__[rubocop]__
* Metrics/AbcSize: Assignment Branch Condition size for output_list is too high. [41.77/30] [:ref](http://c2.com/cgi/wiki?AbcMetric)
* Metrics/PerceivedComplexity: Perceived complexity for output_list is too high. [13/11]